### ISSUE TITLE
BUG: make dist fails because of lzma/C/Makefile.am typo

### ISSUE THREAD

**Issue by pete4abw** [ckolivas/lrzip#173]
I don't know anyone who ever used `make dist`, but in case you try, this small typo will break it!
```
diff --git a/lzma/C/Makefile.am b/lzma/C/Makefile.am
index 74beb93..2f6649d 100644
--- a/lzma/C/Makefile.am
+++ b/lzma/C/Makefile.am
@@ -24,7 +24,7 @@ noinst_LTLIBRARIES = liblzma.la
 # to prevent an error even if -j## is used.
 liblzma_la_SOURCES = \
        $(C_S) \
-       7z.Crc.h \
+       7zCrc.h \
        LzmaDec.h \
        LzmaEnc.h \
        LzFind.c \
```
$ make dist
make  dist-bzip2 dist-gzip am__post_remove_distdir='@:'
make[1]: Entering directory '/tmp/lrzip.631'
make  distdir-am
make[2]: Entering directory '/tmp/lrzip.631'
if test -d "lrzip-0.631"; then find "lrzip-0.631" -type d ! -perm -200 -exec chmod u+w {} ';' && rm -rf "lrzip-0.631" || { sleep 5 && rm -rf "lrzip-0.631"; }; else :; fi
test -d "lrzip-0.631" || mkdir "lrzip-0.631"
 (cd lzma && make  top_distdir=../lrzip-0.631 distdir=../lrzip-0.631/lzma \
     am__remove_distdir=: am__skip_length_check=: am__skip_mode_fix=: distdir)
make[3]: Entering directory '/tmp/lrzip.631/lzma'
make  distdir-am
make[4]: Entering directory '/tmp/lrzip.631/lzma'
 (cd C && make  top_distdir=../../lrzip-0.631 distdir=../../lrzip-0.631/lzma/C \
     am__remove_distdir=: am__skip_length_check=: am__skip_mode_fix=: distdir)
make[5]: Entering directory '/tmp/lrzip.631/lzma/C'
make  distdir-am
make[6]: Entering directory '/tmp/lrzip.631/lzma/C'
make[6]: *** No rule to make target '7z.Crc.h', needed by 'distdir-am'.  Stop.
make[6]: Leaving directory '/tmp/lrzip.631/lzma/C'
make[5]: *** [Makefile:533: distdir] Error 2

**Comment by ckolivas** [comment]
No git pull on this one? This would be an easy one to accept.

**Comment by pete4abw** [comment]
Fixed #176
